### PR TITLE
feat: adds spending limits page

### PR DESCRIPTION
**Pull request by malcodeman**

### CODE REVIEW

File: versioned_docs/version-3.0/dashboard/paymasterRules.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
-# Paymaster Policies
+# Paymaster Rules
 
 Before using a Paymaster to sponsor a transaction you will need to register your smart contracts and white list the methods that will be called in your dApp.
 
 ## Register a Smart Contract
 
-Navigate to the Paymaster you have created, then click policies and contract. If this is the first time you are adding a smart contract the page will look like this:
+Navigate to the Paymaster you have created, then click rules and contract. If this is the first time you are adding a smart contract the page will look like this:
```
Comment (NunoFilipeSantos) [inline versioned_docs/version-3.0/dashboard/paymasterRules.md:12]:
```suggestion
Navigate to the Paymaster you have created, then click `Rules` and `Contract`. If this is the first time you are adding a smart contract the page will look like this:
```

File: versioned_docs/version-3.0/dashboard/spendingLimits.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- Cycle duration (minimum 1 hour, maximum 23 hours)
+
+After clicking Create button the limit will be created.
+
+![add global limit](../images/limits/global_limit_modal.png)
+
+## Add a user limit
+
+Click on Add limit button inside User Limit card and the new user limit modal will appear.
+Both the global and the user limit modals are very simiar and offer same options.
```
Comment (NunoFilipeSantos) [inline versioned_docs/version-3.0/dashboard/spendingLimits.md:26]:
```suggestion
# Add a global limit

Navigate to the Paymaster you have created, then click `Rules` and `Spending`. 
If this is the first time you are adding a limit the page will look like this:

![new limit](../images/limits/new_limit.png)

Click on `Add limit ` inside the Global Limit card and the new global limit modal will appear.
Inside the modal you will be presented with the option of choosing:

- Threshold type (Gas Spend | Number of user operations)
- Threshold value
- Cycle duration (minimum 1 hour, maximum 23 hours)

After clicking `Create` the limit will be created.

![add global limit](../images/limits/global_limit_modal.png)

## Add a user limit

Click on `Add limit` inside User Limit card and the new user limit modal will appear.
Both the global and the user limit modals are very similar and offer the same options.
```

**Review by NunoFilipeSantos — CHANGES_REQUESTED** [review]
Some corrections needed.

**Review by NunoFilipeSantos — DISMISSED** [review]
LGTM now. 😃

File: docs/dashboard/spendingLimits.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+sidebar_label: "Spending Limits"
+sidebar_position: 4
+---
+
+# Add a global limit
```
Comment (himanshugarg06) [inline docs/dashboard/spendingLimits.md:6]:
```suggestion
# Spending limits
This feature allows you to set limits to the gas usage. You can define global limits or per-user quotas, specifying either transaction count or total gas consumption.

## Add a global limit
```

**Review by himanshugarg06 — CHANGES_REQUESTED** [review]
left one comment

**Review by himanshugarg06 — DISMISSED** [review]

File: docs/dashboard/spendingLimits.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+Inside the modal you will be presented with the option of choosing:
+
+- Threshold type (Gas Spend | Number of user operations)
+- Threshold value
+- Cycle duration (minimum 1 hour, maximum 23 hours)
+
+After clicking `Create` the limit will be created.
+
+Please note that whenever limit policy is edited (duration / threshold / threshold type), cycle is reset, and the new cycle starts from the moment of edit.
+Pausing/Activating a limit will not reset the cycle.
```
Comment (NunoFilipeSantos) [inline docs/dashboard/spendingLimits.md:27]:
```suggestion
Please note that whenever the limit policy is edited (duration / threshold / threshold type), the cycle is reset, and the new cycle starts from the moment of edit.
Pausing/Activating a limit will not reset the cycle.
```

**Review by himanshugarg06 — APPROVED** [review]

**Pull request merged**